modules/cnf-about-ptp-and-clock-synchronization.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-about-ptp-and-clock-synchronization_{context}"]
+= About PTP and clock synchronization error events
+
+Cloud native applications such as virtual RAN require access to notifications about events that are critical to the functioning of the overall network. Fast event notifications are early warning signals about impending and real-time PTP clock synchronization events. PTP clock synchronization errors can negatively affect the performance and reliability of your low latency application, for example, a vRAN application running in a distributed unit (DU) environment.
+
+Loss of PTP synchronization is a critical error for a RAN network. If synchronization is lost on a node, the radio may be shut down and the network OTA traffic may be shifted to another node in the wireless network. Fast event notifications mitigates against these workload errors by allowing cluster nodes to communicate PTP clock sync status to the vRAN application running in the DU.
+
+Fast event notifications are generated by the PTP Operator in {product-title} for every PTP capable network interface. The events are made available using a PTP Operator sidecar container over an AMQP message bus. The PTP Operator sidecar container is deployed automatically when the PTP Operator is installed. Event notifications are available to RAN applications that are running on the same DU node. Events notifications are passed to the messaging bus using a REST API operating in a publish/subscribe message architecture. Publish/subscribe messaging, or pub/sub messaging, is an asynchronous service to service communication architecture where any message published to a topic is immediately received by all of the subscribers to the topic.
```

Comment:
The PTP Operator sidecar container is deployed automatically when the PTP Operator is installed.
This is not the case, by default it is not enabled . you need to enable it via ptpOperatorConfig after deployed
``
ptpEventConfig:
    enableEventPublisher: true
    transportHost: "amqp://router.router.svc.cluster.local"
    ``